site/specs/numbers.json

```diff
@@ -1590,29 +1591,30 @@
                   }
                 }
               }
             }
           },
           "204": {
-            "description": "<p>No results matching the specified query parameters were found.</p>\n"
+            "description": "No results matching the specified query parameters were found."
           }
         }
       },
       "post": {
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Creates a bulk port-in order to be used as a template for a collection of child port-in orders.  The template values will be cascaded to child port-ins that result from decomposing a collection of Telephone Numbers that span carriers, RespOrgs, or have attributes that drive the decomposition into a number of individual port-in orders.\nUpon a successfully-submitted payload, the order will have a status of \"DRAFT\", denoting that further modification to the template is expected.  For example, the next step is to use the /tnList endpoint to add a collection of telephone numbers to the bulk port-in order.\nThe only valid value for the ProcessingStatus element in a POST is 'DRAFT', which is the default value.</p>\n",
-        "operationId": "POST /accounts/{accountId}/bulkPortins",
+        "description": "<p>Creates a bulk port-in order to be used as a template for a collection of child port-in orders.  The template values will be cascaded to child port-ins that result from decomposing a collection of Telephone Numbers that span carriers, RespOrgs, or have attributes that drive the decomposition into a number of individual port-in orders.<p></p> Upon a successfully-submitted payload, the order will have a status of \"DRAFT\", denoting that further modification to the template is expected.  For example, the next step is to use the /tnList endpoint to add a collection of telephone numbers to the bulk port-in order. The only valid value for the ProcessingStatus element in a POST is 'DRAFT', which is the default value.</p><p>Please visit <a href='/docs/numbers/guides/bulkportins'>Guides and Tutorials</a></p>",
```

Comment:
```suggestion
        "description": "Creates a bulk port-in order to be used as a template for a collection of child port-in orders.  The template values will be cascaded to child port-ins that result from decomposing a collection of Telephone Numbers that span carriers, RespOrgs, or have attributes that drive the decomposition into a number of individual port-in orders. Upon a successfully-submitted payload, the order will have a status of \"DRAFT\", denoting that further modification to the template is expected.  For example, the next step is to use the /tnList endpoint to add a collection of telephone numbers to the bulk port-in order. The only valid value for the ProcessingStatus element in a POST is 'DRAFT', which is the default value. Please visit <a href='/docs/numbers/guides/bulkportins'>Guides and Tutorials</a>",
```

Reply:
I was going to go through and remove more of the instances of <p> tags but really with a little regex they could all be removed in one operation so I won't list them out individually.